### ISSUE TITLE
Oxygen Harmonization: Code Coverage

### ISSUE THREAD

**Issue by yamahito** [xspec/xspec#147]
Code coverage requires Saxon PE or EE features; however, for many of us our access to those features are as part of the license we get in Oxygen.

Therefore we need to be able to access the code coverage features through the oxygen framework version of XSpec.

This should be possible by supplying Jeni's coverage class as a library to the XSpec transformation (possibly adding a target or property to the build.xml)

**Comment by yamahito** [comment]
Suggested label: feature request

**Comment by cirulls** [comment]
Thanks for this feature request, I agree it would be good to add it so that it can be used from oXygen. As you suggested, I think this would require an additional target in the [ant build file](https://github.com/xspec/xspec/blob/master/build.xml), that's the file used by oXygen to build XSpec.

**Comment by AirQuick** [comment]
I have an initial implementation for this. (But it depends on #187.)

**Comment by AirQuick** [comment]
@yamahito 
I submitted #279 for this.
Please take a look at it and let us know your thoughts, if it still interests you.

**Comment by cirulls** [comment]
@yamahito: pull request #279 is now merged into `master`. Code coverage support is available via Ant and can be run via oXygen. See the pull request for details about the oXygen configuration (at least until the next version of XSpec is integrated into the next version of oXygen).
